pubspec.yaml

```diff
@@ -39,11 +40,8 @@ flutter:
   # included with your application, so that you can use the icons in
   # the material Icons class.
   uses-material-design: true
-
-  # To add assets to your application, add an assets section, like this:
-  # assets:
-  #  - images/a_dot_burr.jpeg
-  #  - images/a_dot_ham.jpeg
+  assets:
+    - assets/images/
```

Comment:
Afin d'utiliser des icones, images, fichiers de config... il les spécifier (fichier/dossier) dans le fichier pubspec.yaml
https://flutter.dev/docs/development/ui/assets-and-images